modules/ui/tools/rapid_internal_features.js

```diff
@@ -1,42 +1,36 @@
 import _debounce from 'lodash-es/debounce';
```

Comment:
Don't worry about the file rename here- I simply noticed that the now-defunct 'fb_roads_toggle' tool from v2 never got pruned from the v3 code. So, I repurposed it since we needed a new tool and renamed it.